script/github-actions/daily-product-scan/tests/mocks/applications/app-1/containers/ConfirmationPage.jsx

```diff
@@ -102,8 +102,8 @@ export class ConfirmationPage extends React.Component {
           </p>
           <p>
             Please don’t apply again. Instead, please call our toll-free hotline
-            at <va-telephone contact="877-222-8387" />. We’re here Monday
```

Comment:
you can actually import this number from a constants file that is maintained in the component-library, which I think is the 'official' way

``` js
import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';

// usage
<va-telephone contact={CONTACTS['222_VETS']} />
```

Reply:
Oh weird, I didn't intend to change those and didn't catch it in the diff. I wonder if my IDE did something funky with formatting. I'll fix it in a commit shortly. Thanks!